parlai/core/torch_ranker_agent.py

```diff
@@ -176,6 +177,27 @@ def __init__(self, opt, shared=None):
                 self.model, device_ids=[self.opt['gpu']], broadcast_buffers=False
             )
 
+    def interactive_mode(self, mode, shared=False):
+        if mode:
+            if not shared:
+                # Only print in the non-shared version.
+                print("[" + self.id + ': full interactive mode on.' + ']')
+            self.eval_candidates = 'fixed'
+            self.ignore_bad_candidates = True
+            self.encode_candidate_vecs = True
+            if self.opt['fixed_candidates_path'] is None:
+                # Attempt to get a standard candidate set for the given task
```

Comment:
since we're leaving this as a TODO for now, can you throw an error here that explains that you  need to specify the fixed candidates file?

Reply:
this is WIP, i plan to fix it before merge